src/components/OptionsSelector.js

```diff
@@ -206,10 +213,16 @@ class OptionsSelector extends Component {
                     <TextInput
                         ref={el => this.textInput = el}
                         value={this.props.value}
-                        onChangeText={this.props.onChangeText}
+                        onChangeText={(text) => {
+                            this.textInput.setNativeProps({selection: null});
+                            this.props.onChangeText(text);
+                        }}
                         onKeyPress={this.handleKeyPress}
                         placeholder={this.props.placeholderText
                             || this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
+                        onBlur={e => this.relatedTarget = e.relatedTarget}
+                        onFocus={() => this.relatedTarget = null}
```

Comment:
Why do we need this?

Reply:
> Why do we need this?


`onBlur` - we store the reference to the element `textInput` lost focus to (to later check if it was an `OptionRow`)
`onFocus` - we delete that reference since `textInput` was focused an we are not interested in the previous relatedTarget

Reply:
It is kind of unnecessary but I am fine with it. let me know when PR is updated

Reply:
> It is kind of unnecessary but I am fine with it.

You're right, it is unnecessary, updated the PR